src/openzaak/config/forms.py

```diff
@@ -43,3 +48,41 @@ def __init__(self, *args, **kwargs):
 
         if instance and instance.component == "autorisaties":
             self.fields["enabled"].disabled = True
+
+
+def get_nlx_choices() -> Dict[str, dict]:
+    choices = {}
+    nlx_outway = NLXConfig.get_solo().outway
+    try:
+        services = get_services()
+    except Exception:
```

Comment:
a logger call here wouldn't be bad, pokemon-exception catching is a bad practice :D